docs/introduction.md

```diff
@@ -0,0 +1,54 @@
+# Introduction
+
+## What is BaNaNaS?
+
+BaNaNaS is OpenTTD's content service.
+It is the distribution platform OpenTTD uses to allow content-creators to offer additions to OpenTTD and users to use them in their games.
+
+This services was created in 2007, and has been extended a few times over the years.
+It currently offers the following capabilities:
+
+- Uploading of new content by content-creators
+- Editing / updating content by the original content-creator
+- Downloading this content in-game by any user
+- Browsing on the web what content is available
+
+This can be done for the following content-type in OpenTTD:
+
+- Base Graphics
```

Comment:
I can't remember how long we discussed the proper spelling of the content types. But iirc the result was to use hypens.

Reply:
I thought we decided not to use hyphens. :P